### PR TITLE
Updated docs screenshots, text, and styling

### PR DESCRIPTION
**Pull request by garethx**
## What kind of change does this PR introduce?

Updates screenshots in documentation to be more consistent with the marketing site. Tweak a couple of CSS bits to update headers to use the design system blue, also tweak some copy to update it with recent changes.

@DannyRuchtie See https://pr3834.build.csb.dev/docs for design review. This is only _some_ of the screenshots, as we don't have figma assets for all screens. We should discuss how to handle those, but this is a start at least.

## Checklist
- [x] Testing
- [x] Ready to be merged - pending design review

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3834/commits/8f414f4ef6c95db31639f921418cf0f043dbc6c3 is at https://pr3834.build.csb.dev/s/new.

File: packages/homepage/content/docs/2-live.md [hunk 1]
```diff
@@ -38,10 +36,4 @@ You can enable Classroom Mode from the Live Mode drop-down menu.
 
 ![Classroom mode](./images/live-classroom.png)
 
-You can give someone editor rights by pressing the '+' icon next to their name.
-
-![Going Live](./images/live-make-editor.png)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:43]: Don't forget to delete this image

File: packages/homepage/content/docs/2-live.md [hunk 2]
```diff
@@ -38,10 +36,4 @@ You can enable Classroom Mode from the Live Mode drop-down menu.
...
-
-You can make someone a spectator by clicking the '-' icon next to their name.
-
-![Going Live](./images/live-make-spectator.png)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:47]: Same

File: packages/homepage/content/docs/5-secrets.md [hunk 3]
```diff
@@ -30,12 +30,10 @@ Pro tip: It's good practice to name your secrets all in uppercase.
 After clicking the "Add Secret" button the secret is added, the sandbox is
 restarted, and you can see the list of all your secrets above the form.
 
-![Secrets](./images/secrets-3.png)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/5-secrets.md:33]: Same

File: packages/homepage/content/docs/5-secrets.md [hunk 4]
```diff
@@ -48,9 +46,7 @@ secrets in your server-side code.
 ## Editing secrets
 
 You can edit and remove existing secrets. Click on the pencil to edit the name
-and value of your secret:
-
-![Icons](./images/secrets-4.png)
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/5-secrets.md:53]: Same

**Comment by DannyRuchtie** [comment]
Looking good @garethx

File: packages/homepage/content/docs/2-live.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ description:
 
 ## What is Live?
 
-Live is a mode in CodeSandbox that allows you to share your sandbox with others
+All sandboxes on CodeSandbox are live, meaning you can share your sandbox with others
 for real time collaboration on code. You can simultaneously create, delete, edit
 and move files and code with others in a sandbox, as well as chat about your
 changes. It's like Google Docs, but for coding.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:13]:
```suggestion
All sandboxes on CodeSandbox are live, meaning you can share your sandbox with
others for real time collaboration on code. You can simultaneously create,
delete, edit for real time collaboration on code. You can simultaneously create,
delete, edit and move files and code with others in a sandbox, as well as chat
about your and move files and code with others in a sandbox, as well as chat
about your changes. It's like Google Docs, but for coding. changes. It's like
Google Docs, but for coding.
```

File: packages/homepage/content/docs/2-live.md [hunk 6]
```diff
@@ -16,9 +16,7 @@ changes. It's like Google Docs, but for coding.
 
 ## Going Live
 
-You can go live by pressing the 'Go Live' button under the Live tab in the
-left-hand activity bar in the editor. This will give you a URL that you can
-share with others to join the session. Invited editors can only make edits to
+All sandboxes are live by default, you can invite contributors to join the sandbox using the share button from the top bar in the editor. In addition, if you go to the Live tab from the activity bar, you can get a Live URL to share with guests (non-members of the sandbox) so they can temporarily join the session. Guest editors can only make edits to
 the sandbox while they are in the live session, they cannot do this when just
 viewing the sandbox.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:21]:
```suggestion
All sandboxes are live by default, you can invite contributors to join the
sandbox using the share button from the top bar in the editor. In addition, if
you go to the Live tab from the activity bar, you can get a Live URL to share
with guests (non-members of the sandbox) so they can temporarily join the
session. Guest editors can only make edits to the sandbox while they are in the
live session, they cannot do this when just viewing the sandbox.
```

File: packages/homepage/content/docs/2-live.md [hunk 7]
```diff
@@ -38,10 +36,4 @@ You can enable Classroom Mode from the Live Mode drop-down menu.
...
 ![Classroom mode](./images/live-classroom.png)
 
-You can give someone editor rights by pressing the '+' icon next to their name.
-
-![Going Live](./images/live-make-editor.png)
-
-You can make someone a spectator by clicking the '-' icon next to their name.
-
-![Going Live](./images/live-make-spectator.png)
+You can give someone editor rights by pressing the '+' icon next to their name, or you can make someone a spectator by clicking the '-' icon next to their name.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/2-live.md:39]:
```suggestion
You can give someone editor rights by pressing the '+' icon next to their name,
or you can make someone a spectator by clicking the '-' icon next to their name.
```

File: packages/homepage/content/docs/5-secrets.md [hunk 8]
```diff
@@ -30,12 +30,10 @@ Pro tip: It's good practice to name your secrets all in uppercase.
 After clicking the "Add Secret" button the secret is added, the sandbox is
 restarted, and you can see the list of all your secrets above the form.
 
-![Secrets](./images/secrets-3.png)
-
 Secrets are environment variables, meaning they are defined on `process.env`. In
 the example above, we can read the API key from
 `process.env.GOOGLE_MAPS_API_KEY`. See the example below, showing how to access
-secrets in your server-side code.
+secrets in your server-side code (though obviously you wouldn't want to share a secret like this, it justs illustrates how to access them).
```
Comment (MichaelDeBoey) [inline packages/homepage/content/docs/5-secrets.md:36]:
```suggestion
secrets in your server-side code (though obviously you wouldn't want to share a
secret like this, it justs illustrates how to access them).
```

**Pull request merged**